wordpress-mobile/gutenberg-mobile
# Use Gutenberg store

This PR switches the current store with the one provided by gutenberg.

Only `App` and `AppContainer` are updated for now since those deal with the state. Some minor changes on `BlockManager` are necessary since the gb store already deals with merging blocks.

We'll also want to connect some of our components to the store and reuse existing components from the `gutenberg` repository (`BlockList`, `BlockListBlock` and `BlockEdit`)

### Testing Instructions
- `git submodule update --init`
- `yarn install`
- `yarn start:reset`
- Check that there are no important regressions on android and ios (`yarn ios` and `yarn android`)
- Run the tests `bash ./.travis/travis-checks-js.sh`
